ui/tests/helpers/sync/setup-models.js

```diff
@@ -17,7 +17,7 @@ export function setupModels(hooks) {
       id: destination.name,
     });
     this.destination = this.store.peekRecord(destinationModelName, destination.name);
-    this.destinations = this.store.peekAll(destinationModelName).toArray();
+    this.destinations = this.store.peekAll(destinationModelName);
```

Comment:
Even though we're modifying the original array here, this seems find for tests (I presume that's why you didn't use `slice()` here!)